src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -419,6 +419,16 @@ extension SecureStore: PrivacyPreservingProviding {
 		}
 	}
 	
```

Comment:
set the `testResultMetadata` and the `dateOfConversionToHighRisk` to nil in the setter of the `isPrivacyPreservingAnalyticsConsentGiven ` is missing

Reply:
the dateOfConversionToHighRisk is not only related to this feature, we may need it for other features in the future

Reply:
Perhaps then we should remove it from PrivacyPreservingProviding

Reply:
As long it is only used in the context of the PPA, then I would let it where it is.